frankenstein/cmd/frankenstein/main.go

```diff
@@ -205,9 +219,17 @@ func setupIngester(
 		log.Fatal(err)
 	}
 	prometheus.MustRegister(ingester)
-	http.Handle("/push", frankenstein.AppenderHandler(ingester))
-	http.Handle("/query", frankenstein.QueryHandler(ingester))
-	http.Handle("/label_values", frankenstein.LabelValuesHandler(ingester))
+
+	instr := middleware.Merge(
+		middleware.Logging,
```

Comment:
Do we really want to log every sample push?  How about only tracking those via metrics?

Also, I wonder if it's worth pulling in a 2700+ line dependency (gorilla and Weaveworks dep) just for a small bit of instrumentation.